File: submissions/sm_001_aalind/week_03/day_3/session_2/average_divisible_while.js

function average_divisible_while(limit, num) {
  var sum = 0;
  var count = 0;
  var i = 1;

  while(i <= limit) {
    if(i % num == 0) {
      sum = sum + i;
      count = count + 1;
    }
    i = i + 1;
  }

  return (sum/count);
}

console.log(average_divisible_while(20, 3));

File: submissions/sm_001_aalind/week_13/day_4/session_1/average_divisible_while.py
limit = int(input("Enter limit: "))
num = int(input("Enter divisor: "))

start = 1
sum = 0
count = 0

while start <= limit:
    if start % num == 0:
        sum += start
        count += 1
    start += 1

print(sum/count)